data/css/modules/layout/_sidebar.scss

```diff
@@ -6,57 +6,49 @@
 
         .CardTitle {
             margin: 0px;
-            padding: 6px;
-            border: 2px solid transparentize(white, 1);
-            background-color: transparentize(white, 1);
-            transition: background-color 250ms ease-in-out,
-                        background-image 250ms ease-in-out,
-                        border 250ms ease-in-out;
+            padding: 10px;
+            // border-bottom: 1px solid transparentize(black, 1 - 0.7);
 
             &__title {
                 padding: 0em 0em 0em 1.5em;
                 font-family: $display-font;
+                font-weight: 700;
                 font-size: 1.125em; /* 18pt * 1/12 * 0.75 DPI correction factor */
-                font-weight: normal;
                 color: $primary-light-color;
-                transition: color 250ms ease-in-out;
             }
 
             &:hover {
-                border: 2px solid white;
-                background-color: transparentize(white, 1 - 0.9);
-                background-image: linear-gradient(-179deg,
-                                                  transparentize(white, 1 - 0.05) 0%,
-                                                  transparentize(black, 1 - 0.05) 100%);
 
                 .Card__title {
                     color: $primary-medium-color;
                 }
             }
 
+            &:active {
+                background: transparentize(black, 1 - 0.3);
+            }
+
             &.highlighted {
-                background: transparentize(white, 1 - 0.8);
-                text-shadow: 0px 1px 0px white;
-                font-weight: bold;
-                transition-property: none;
+                background: transparentize(black, 1 - 0.5);
+                font-weight: 700;
 
                 .Card__title {
-                    color: $primary-dark-color;
-                    transition-property: none;
+                    color: white;
+                    transition-property: color;
+                    transition-timing-function: ease;
+                    transition-duration: 0.5s;
```

Comment:
Nice! Have we tested perf on the mini or another lower end device for this? Or will it be disabled by default on those devices, I forget how we performance bucket.